### PR TITLE
Modify scenario files to allow for direct input Issue#33

### PR DESCRIPTION
**Pull request by hforeste**
Add to your scenario file a json array of input values
. . .
        "inputPath":  "..\\queueitems-static-64.txt",
        **"input": [64,65]**,
        "triggerType":  "Queue",
. . .

### CODE REVIEW

File: SampleUsages/TestScenario.cs [hunk 1]
```diff
@@ -34,56 +37,51 @@ internal void RunScenario(ILogger logger, ServerlessResultManager.TestScenario d
...
             FunctionTest test = null;
             TriggerTestLoadProfile profile;
-            var inputCount = 0;
+
+            string[] inputItems;
+            try
+            {
+                inputItems = this.Input ?? Directory.GetFiles(this.InputPath);
+            }
+            catch
```
Comment (paulbatum) [inline SampleUsages/TestScenario.cs]: Don't use exceptions for normal control flow. Check to see if the input path is a directory or a file and apply the appropriate logic.

Comment (paulbatum) [inline SampleUsages/TestScenario.cs]: Actually I think I'm just confused about this code. Why the call to Directory.GetFiles(), what scenario is that for?

Comment (hforeste) [inline SampleUsages/TestScenario.cs]: Directory.GetFiles() are for blob scenarios and aws s3 scenarios

Comment (KrzysiekKnigawka) [inline SampleUsages/TestScenario.cs]: I agree on not using exceptions for normal control flow. Once that get fixed I approve.

**Pull request merged**